### ISSUE TITLE
Add aria-label to footer nav element

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#4303]
### Overview

The ARIA specification states that if a page includes more than one navigation landmark, each should have a unique label.  In this issue we will label the footer navigation element to identify it as the secondary navigation, distinguishing it from the primary navigation element in the header.  

### Action Items

- [x] The html for the footer navigation can be found in _includes/footer.html
- [x] Change `<nav class="footer-nav">`
  to:
  `<nav class="footer-nav" aria-label="Secondary">`

### Resources/Instructions

- [Footer Include](https://github.com/hackforla/website/blob/gh-pages/_includes/footer.html)
- [ARIA Authoring Practices Guide - Landmark Regions](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/)
- ARIA 1.2 Specification: [navigation landmark](https://www.w3.org/TR/wai-aria-1.2/#navigation)

**Comment by mademarc** [comment]
Progress:  gonna fix this issue by end of the week  
Blockers: no blockers 
Availability: 3/30/2023
ETA: 4/4/2023

**Comment by amath95** [comment]
Availability: 04/09
ETA: EOD

**Comment by StephenTheDev1001** [comment]
Availability: Apr 3rd - 7th evenings PDT.
ETA: EOD
